test/operators.jl

```diff
@@ -339,3 +339,11 @@ end
     Base.:(<)(::B46327, ::B46327) = false
     @test B46327() <= B46327()
 end
+
+@testset "getproperty/hasproperty" begin
+    gp = Base.Fix2(Base.getproperty, :x)
+    @test Base.getproperty(:x) == gp
+    @test Base.hasproperty(:x) == Base.Fix2(Base.hasproperty, :x)
+    @test Base.getproperty(:x) == gp
+    @test Base.hasproperty(:x)(gp)
```

Comment:
What is this line supposed to test?

Reply:
Similar to the above (now fixed) test it's intended to test the evaluation of `hasproperty(gp, :x)`. I was mostly trying to be thorough with the testing, but if it's just redundant I can remove these last two

Reply:
No, it's not redundant. It would be appreciated if you make test cases for anything added newly.